src/sfizz/Macros.h

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: BSD-2-Clause
+
+// This code is part of the sfizz library and is licensed under a BSD 2-clause
+// license. You should have receive a LICENSE.md file along with the code.
+// If not, contact the sfizz maintainers at https://github.com/sfztools/sfizz
+
+#pragma once
+
+#define UNUSED(x) (void)(x)
+
+#if defined SFIZZ_EXPORT_SYMBOLS
+  #if defined _WIN32
+    #define SFIZZ_EXPORTED_API __declspec(dllexport)
+  #else
+    #define SFIZZ_EXPORTED_API __attribute__ ((visibility ("default")))
+  #endif
+#else
+  #define SFIZZ_EXPORTED_API
+#endif
+
+#if __cplusplus > 201103L
+#define CONSTEXPR_OR_INLINE constexpr
+#else
+#define CONSTEXPR_OR_INLINE inline
+#endif
```

Comment:
Why not let it be inline in both cases?

Reply:
constexpr is more useful no? with inline I do not think the compiler will deduce the function and remove it

Reply:
`constexpr` means for the function to be evaluable at *either* run-time or compile-time.
If the former case, it should be treated as with inline tag, as with the older compiler. (if only for the sake of consistency)

Reply:
> A constexpr specifier used in a function or static member variable (since C++17) declaration implies inline.

[Ref](https://en.cppreference.com/w/cpp/language/constexpr) . But maybe your comment is that we could use `inline CONSTEXPR` with a `CONSTEXPR` macro that removes itself on cpp 11?

Reply:
>A constexpr specifier used in a function or static member variable (since C++17) declaration implies inline

C++17 and up

>Ref . But maybe your comment is that we could use inline CONSTEXPR with a CONSTEXPR macro that removes itself on cpp 11?

It's the idea yes, making a function constexpr only for 14 and up.

Reply:
The c++17 part seems only for member variables but I may be misreading 😄

Reply:
Anyway if you find it clearer I have no qualms. The macro could be CXX14_CONSTEXPR?

Reply:
yes